Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-SolarStorm_and_SUNBURST_Hunting_and_Response_Playbook_6.yml

```diff
@@ -2112,6 +2112,7 @@ tasks:
       id: efb2c07f-c31b-478d-8074-460011014ec8
       version: -1
       name: Found flows relate to SolarWinds?
+      description: Checks if found any risky flows relate to SolarWinds
```

Comment:
name: Were risky flows related to SolarWinds found?
      description: Checks if any risky flows related to SolarWinds were found